app-tasks/dags/export/do_export.py

```diff
@@ -0,0 +1,133 @@
+from __future__ import print_function
+import datetime
+import logging
+import os
+import time
+import json
+
+import subprocess
+
+from airflow.models import DAG
+from airflow.operators import BashOperator, PythonOperator
+from airflow.exceptions import AirflowException
+
+from rf.utils.io import get_session
+from rf.utils.exception_reporting import wrap_rollbar
+
+
+rf_logger = logging.getLogger('rf')
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+ch.setFormatter(formatter)
+rf_logger.addHandler(ch)
+
+HOST = os.getenv('RF_HOST')
+API_PATH = '/api/exports/'
+
+logger = logging.getLogger(__name__)
+
+default_args = {
+    'owner': 'raster-foundry',
+    'start_date': datetime.datetime(2017, 1, 1)
+}
+
+dag = DAG(
+    dag_id='do_export',
+    default_args=default_args,
+    schedule_interval=None,
+    concurrency=os.getenv('AIRFLOW_DAG_CONCURRENCY', 4)
+)
+
+
+
+# UTILS
+@wrap_rollbar
+def update_status(export, status):
+    url = '{0}{1}{2}/'.format(HOST, API_PATH, export)
```

Comment:
I think that _is_ something holding on to -- I wouldn't want one bad export or a time-consuming one to prevent progress on other ones necessarily.

The end goal is to reduce our dependency on the `rf` library in here as much as possible, hitting an endpoint here and there, probably isn't too bad. Could also add an additional function in the `jar` that would list pending exports, output them, and then parse/split on whitespace/commas too.